packages/core/src/i18n.test.ts

```diff
@@ -290,6 +290,25 @@ describe("I18n", () => {
     expect(handler).toHaveBeenCalledTimes(2)
   })
 
+  it("._ should emit missing event for undefined id", () => {
+    const i18n = setupI18n({
+      locale: "en",
+      messages: { en: {} },
+    })
+
+    const messageID = undefined
+
+    const handler = jest.fn()
+    i18n.on("missing", handler)
+    // @ts-ignore
+    i18n._(messageID)
```

Comment:
if it's exepcted to have a TS error here, put `@ts-expect-error` with explanation why it's expected.

Reply:
changed to 
// @ts-expect-error 'id' should be of 'MessageDescriptor' or 'string' type.